src/views/Workflow.vue

```diff
@@ -52,37 +52,31 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 </template>
 
 <script>
```

Comment:
This file will have mainly code being removed. This is because the behavior of views & components with subscriptions has been standardized, and is now in mixins and in the Workflow Service. I will write a couple paragraphs on how to add a view with a widget here, and use that later to update the Table view PR.